site/docs/components/number-input/examples.mdx

```diff
@@ -10,118 +10,122 @@ data:
 
 ## Default
 
-Number input has an increment step value of 1 and a step block value of 10, and you can only use it to accept numeric characters.
+The `NumberInput`, by default, features an increment `step` of 1, enabling precise value adjustments. For larger value adjustments, users can utilize the `Shift` key in combination with the `Arrow` keys to apply a `stepMultiplier` which is 2 by default.
```

Comment:
I don't think we typically use backticks this way e.g. `Shift`, `Arrow`. Typically they are reserved for component names, prop names and literal values.